lib/impl/jacoco.js

```diff
@@ -0,0 +1,74 @@
+(function (jacocoParse, Promise, Joi, logger, util, path) {
+    'use strict';
+
+    var jacocoStringValidation = Joi.string().required();
+    var optionsValidation = Joi.object().keys().optional();
+
+    module.exports = {
+        parse: function parseJacoco(pathPrefix, jacocoString, options) {
+            return new Promise(function (resolve, reject) {
+                logger.debug('Parsing Jacoco Data');
+                var validLcov = Joi.validate(jacocoString, jacocoStringValidation);
```

Comment:
`validLcov` seems weird here. I would prefer `nonEmptyReport` report since it is kind of the check is doing.

Reply:
I changed the variable name to nonEmptyReport from validJacoco.